docs/doxygen/include/size_table.md

```diff
@@ -9,8 +9,8 @@
     </tr>
     <tr>
         <td>sigv4.c</td>
-        <td><center>4.9K</center></td>
-        <td><center>4.2K</center></td>
+        <td><center>5.1K</center></td>
+        <td><center>4.3K</center></td>
```

Comment:
Is the user likely to have already hashed the payload and encoded it in lowercase hex? Is the extra memory usage worth the speedup gained from not having to hash the payload?

Reply:
Yeah, In almost all cases the payload will be hash and hex encoded by the library. So I think it is worth.

Reply:
But is that header something the application is expected to have been added already, or would a user expect the library to add the header for them? If it's the latter case, then the user can save themself time by just telling the library to add it for them

Reply:
The application will add that header before calling the Sigv4 API.

Reply:
So it's not almost all cases then, it's every case